### PR TITLE
added max_instance_lifetime

### PR DESCRIPTION
**Pull request by vglen**
# PR o'clock

## Description

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [x] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [ ] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Next release
 
-## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.1.0...HEAD)] - YYYY-MM-DD]
```
Comment (max-rocket-internet) [inline CHANGELOG.md]: Please leave this line unchange

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 repos:
-- repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.22.0
-  hooks:
-    - id: terraform_fmt
-    - id: terraform_docs
-      args: [--args=--with-aggregate-type-defaults --no-escape]
```
Comment (max-rocket-internet) [inline .pre-commit-config.yaml]: We need these args but please do update the version 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (max-rocket-internet) [reply CHANGELOG.md]: Ahh it looks like @barryib in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/714

File: .pre-commit-config.yaml (hunk 2 above)
Comment (vglen) [reply .pre-commit-config.yaml]: ok, any reason why the terraform-dcos is failing? I would also run the tests, but running into ruby version hell. I need to get rvm going.

File: CHANGELOG.md (hunk 1 above)
Comment (vglen) [reply CHANGELOG.md]: I removed that from my changelog.md and put it back to match master.

**Comment by barryib** [comment]
Can you please update your branch and upgrade to terraform-docs 0.8.1 ?

**Comment by barryib** [comment]
Closing. Resolved by https://github.com/terraform-aws-modules/terraform-aws-eks/pull/769/files

**Pull request closed without merging**